src/commands/vote.ts

```diff
@@ -107,7 +108,7 @@ async function main(interaction: ChatInputCommandInteraction) {
     );
 
     const reply = await wantsToStartVote.followUp(
-      `**${topic}**\n*by ${interaction.member}, ends on ${date}*\n\n${optionsString}`,
+      `**${topic}**\n*by ${interaction.member}, ends on ${formatAsTimestamp(date, TimestampType.LONG_DATE_TIME)}*\n\n${optionsString}`,
```

Comment:
Just had the idea that a relative timestamp might even make more sense here? If so, let's also remove the "on" since i think it will take care of that